### ISSUE TITLE
Does this library support TWiML messages?

### ISSUE THREAD

**Issue by 0syntrax0** [sfreiberg/gotwilio#88]
I've looked in the code, but I have not found anything useful to send TWiML messages https://www.twilio.com/docs/sms/twiml/message .

Just asking before I create a PR to add the functionality.

**Comment by lryong** [comment]
@sfreiberg  Yes, I had this problem too. Can this library include TWiML messages, although I use [the official library](https://github.com/twilio/twilio-go), but it is not as good as this library.

**Comment by ozfive** [comment]
I actually need this myself!
